include/vcpkg/binarycaching.h

```diff
@@ -77,12 +78,15 @@ namespace vcpkg
 
         LocalizedString valid();
         std::string instantiate_variables(const Dependencies::InstallPlanAction& action) const;
+        std::string get_url(BinaryConfigParserState state);
```

Comment:
```suggestion
```

There doesn't seem to be any reason for this one to be a member and it can live in the tests just like the others in `configparser.cpp`.